Parsers/ASimRegistryEvent/test/VMwareCarbonBlackCloud_ASimRegistryEvent_SchemaTest.csv

```diff
@@ -0,0 +1,43 @@
+﻿Result
+"(1) Warning: Missing recommended field [ActorUserId]"
+"(1) Warning: Missing recommended field [RegistryPreviousKey]"
+"(1) Warning: Missing recommended field [RegistryPreviousValueData]"
+"(1) Warning: Missing recommended field [RegistryPreviousValueType]"
+"(1) Warning: Missing recommended field [RegistryPreviousValue]"
+"(1) Warning: Missing recommended field [RegistryValueData]"
+"(1) Warning: Missing recommended field [RegistryValueType]"
+"(1) Warning: Missing recommended field [RegistryValue]"
+"(2) Info: Missing optional alias [Rule] aliasing non-existent column [RuleName]"
+"(2) Info: Missing optional field [ActingProcessGuid]"
+"(2) Info: Missing optional field [ActorSessionId]"
+"(2) Info: Missing optional field [DstDescription]"
+"(2) Info: Missing optional field [DvcDescription]"
+"(2) Info: Missing optional field [DvcMacAddr]"
+"(2) Info: Missing optional field [DvcOsVersion]"
+"(2) Info: Missing optional field [DvcScopeId]"
+"(2) Info: Missing optional field [EventOriginalSubType]"
+"(2) Info: Missing optional field [EventProductVersion]"
+"(2) Info: Missing optional field [EventReportUrl]"
+"(2) Info: Missing optional field [ParentProcessGuid]"
+"(2) Info: Missing optional field [RuleName]"
+"(2) Info: Missing optional field [RuleNumber]"
+"(2) Info: Missing optional field [SrcDescription]"
+"(2) Info: Missing optional field [ThreatCategory]"
+"(2) Info: Missing optional field [ThreatConfidence]"
+"(2) Info: Missing optional field [ThreatField]"
+"(2) Info: Missing optional field [ThreatFirstReportedTime]"
+"(2) Info: Missing optional field [ThreatId]"
+"(2) Info: Missing optional field [ThreatIsActive]"
+"(2) Info: Missing optional field [ThreatLastReportedTime]"
+"(2) Info: Missing optional field [ThreatName]"
```

Comment:
@jayeshprajapaticrest @niralishah-crest 
are you planning to generated threat events for registry?

Reply:
We are unable to generate more threat logs as our IT team is unable to resetup VMware Carbon Black Cloud Environment.